diesel/src/sql_types/mod.rs

```diff
@@ -349,6 +349,41 @@ pub struct Time;
 #[diesel(mysql_type(name = "Timestamp"))]
 pub struct Timestamp;
 
+/// The timestamp with timezone SQL type.
+///
+/// ### [`ToSql`](crate::serialize::ToSql) impls
+///
+/// - [`std::time::SystemTime`][SystemTime] (PG only)
+/// - [`chrono::NaiveDateTime`][NaiveDateTime] with `feature = "chrono"`
+/// - [`time::Timespec`][Timespec] with `feature = "deprecated-time"` (PG only)
+///
+/// ### [`FromSql`](crate::deserialize::FromSql) impls
+///
+/// - [`std::time::SystemTime`][SystemTime] (PG only)
+/// - [`chrono::NaiveDateTime`][NaiveDateTime] with `feature = "chrono"`
+/// - [`time::Timespec`][Timespec] with `feature = "deprecated-time"` (PG only)
+///
+/// [SystemTime]: std::time::SystemTime
```

Comment:
I'll admit that I copy and pasted this from the above `Timestamp` type but wasn't quite sure whether I needed to add even more (in particular it looks like there is support for `DateTime<Local | Utc | TimeZone>` objects now) to the documentation